packages/application-generic/src/services/queues/queue-base.service.ts

```diff
@@ -71,52 +71,60 @@ export class QueueBaseService {
     await this.gracefulShutdown();
   }
 
-  public async addMinimalJob(
-    id: string,
-    data?: any,
-    groupId?: string,
-    options: JobsOptions = {}
-  ) {
+  public async addMinimalJob(params: IJobParams) {
+    const { name, groupId, data, options } = params;
+
     const jobData = data
       ? {
           _environmentId: data._environmentId,
-          _id: id,
+          _id: name,
           _organizationId: data._organizationId,
           _userId: data._userId,
         }
       : undefined;
 
-    await this.add(id, jobData, groupId, {
-      removeOnComplete: true,
-      removeOnFail: true,
-      ...options,
-    });
+    await this.instance.add(
+      name,
+      jobData,
+      {
+        removeOnComplete: true,
+        removeOnFail: true,
+        ...options,
+      },
+      groupId
+    );
   }
 
-  public async add(
-    name: string,
-    data?: any,
-    groupId?: string,
-    options: JobsOptions = {}
-  ) {
+  public async add(params: IJobParams) {
```

Comment:
Refactored to get object instead of params list so we could be explicit about the params.